src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/components/dot-layout-designer/dot-layout-designer.component.scss

```diff
@@ -30,16 +30,21 @@
 }
 
 [class^='dot-layout-designer__sidebar'] {
+    align-items: normal;
+    background-color: transparent;
     position: relative;
```

Comment:
This ```position: relative``` now should be inside the new sidebar component because that's the one that needs it to the position absolute of the three dots button.